src/master/quickstart.md

```diff
@@ -134,47 +153,47 @@ immuclient> use mydatabse
 
 ## SQL operations with immuclient
 
-In addition to a key-value store, immudb supports the relational model (SQL). For example, to a table:
+In addition to a key-value store, immudb supports the relational model (SQL). For example, to create a table:
 
-```sh
-$ immuclient exec "CREATE TABLE people(id INTEGER, name VARCHAR, salary INTEGER, PRIMARY KEY id);"
-sql ok, Ctxs: 1 Dtxs: 0
+```bash
+$ ./immuclient exec "CREATE TABLE people(id INTEGER, name VARCHAR, salary INTEGER, PRIMARY KEY id);"
+Updated rows: 0
 ```
 
 To insert data, use `UPSERT` (insert or update), which will add an entry, or overwrite it if already exists (based on the primary key):
```

Comment:
Maybe we should start with a single INSERT statement which is easier to understand (and more common) and only then introduce UPSERT?

Reply:
I think it is fine with upsert, this is just intro that you can do anything with sql.